grpc-client-utils/src/main/java/org/hypertrace/core/grpcutils/client/GrpcChannelRegistry.java

```diff
@@ -30,44 +41,56 @@ public ManagedChannel forSecureAddress(String host, int port) {
   }
 
   public ManagedChannel forSecureAddress(String host, int port, GrpcChannelConfig config) {
-    assert !this.isShutdown;
+    assert !this.isShutdown();
```

Comment:
wow lol

Reply:
hah - this isn't javascript, that wouldn't fly :) but the context on this change: this seemed like a reasonable thing to expose since calling a creation on a shutdown registry would lead to an error (and extending it with a subclass needed this information anyway, so rather than making it protected I properly exposed it).